src/components/Tabs/Tabs.js

```diff
@@ -298,10 +300,16 @@ export default class Tabs extends React.Component<Props, State> {
         >
           <TabWrapper showSVGShadow={isActive && Platform.OS === 'android' && tabLengths[id]} width={tabLengths[id]}>
             <TabItem
+              activeOpacity={1}
               active={isActive}
-              onPress={() => this.setState({
-                activeTab: id,
-              }, onPress)}
+              onPress={() => {
+                if (onTabChange) onTabChange(true);
+                this.setState({ activeTab: id });
+                requestAnimationFrame(() => {
```

Comment:
Maybe this needs to be called after state (active tab ID) has changed or it's ok?

Reply:
You mean, as a callback of setState?
All of those actions should happen at the same time - we set an active tab, and we render view, that is related to the active tab